packages/devextreme/js/__internal/ui/tree_view/m_tree_view.base.ts

```diff
@@ -799,18 +799,6 @@ const TreeViewBase = (HierarchicalCollectionWidget as any).inherit({
     return ['url', 'linkAttr'];
   },
 
-  _addContent($container, itemData) {
```

Comment:
The adaptive menu doesn't have links anymore. I don't think this was expected to be done in this PR. I left a more descriptive comment in the Teams post.

Reply:
Hello i did a slight modification on this issue i passed adaptivityEnabled option to treeview when used by menu i used that to check if its needed to render links.